### DISCUSSION TITLE
Switching to rune-discuss Google group

### DISCUSSION THREAD

**Discussion by waywardgeek** [google/rune#45] [Announcements]
If anyone wants to join in the Rune discussion, feel free to ask to join the rune-discuss Google group at groups.google.com

This is part of our effort to start developing the experimental Rune programming language in the open.  Google groups are more compatible with email systems, so we can, for example, CC someone on a post, or start a thread straight from our email client.

**Comment by 0dminnimda** [comment]
Is this group a private one?

**Reply by avindra** [reply]
> Is this group a private one?

it's open

https://groups.google.com/g/rune-discuss
